File: 백준/Bronze/31669. 특별한 학교 탈출/특별한 학교 탈출.c
#include <stdio.h>
#include <string.h>

int main() {
    int N, M;
    scanf("%d %d", &N, &M);
    
    int lst[M];
    memset(lst, 0, sizeof(lst));
    
    for (int i = 0; i < N; i++) {
        char tmp[M + 1];
        scanf("%s", tmp);
        
        for (int m = 0; m < M; m++) {
            if (tmp[m] == 'O') {
                lst[m]++;
            }
        }
    }
    
    int found = 0;
    for (int m = 0; m < M; m++) {
        if (lst[m] == 0) {
            printf("%d\n", m + 1);
            found = 1;
            break;
        }
    }
    
    if (!found) {
        printf("ESCAPE FAILED\n");
    }

    return 0;
}

File: 백준/Bronze/31669. 특별한 학교 탈출/특별한 학교 탈출.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int N = sc.nextInt();
        int M = sc.nextInt();
        sc.nextLine(); 
        
        int[] lst = new int[M];
        for (int i = 0; i < N; i++) {
            String tmp = sc.nextLine();
            for (int m = 0; m < M; m++) {
                if (tmp.charAt(m) == 'O') {
                    lst[m]++;
                }
            }
        }
        
        boolean found = false;
        for (int m = 0; m < M; m++) {
            if (lst[m] == 0) {
                System.out.println(m + 1);
                found = true;
                break;
            }
        }
        if (!found) {
            System.out.println("ESCAPE FAILED");
        }
        sc.close();
    }
}
